test/model/tools/model_test/main.cpp

```diff
@@ -868,9 +871,10 @@ main(int argc, char *argv[])
 
             /* Generate random timing stress configurations and add it to the WiredTiger config. */
             if (generate_timing_stress_configurations) {
-                std::string rand_env_config;
-                rand_env_config = model::kv_workload_generator::generate_configurations(seed);
-                wt_conn_config = model::join(wt_conn_config, rand_env_config);
+                std::string rand_stress_config;
+                rand_stress_config =
```

Comment:
You can assign straight away or use the string constructor `std::string rand_stress_config(<value>)`
```suggestion
                std::string rand_stress_config =
```

Reply:
Thank you for this suggestion! I like this.